project-base/storefront/pages/products/[productSlug].tsx

```diff
@@ -75,6 +87,17 @@ export const getServerSideProps = getServerSidePropsWrapper(
                     };
                 }
 
+                if (
```

Comment:
Why do you need to set this during SSR? Especially when it is set in browser again...? 🤔

Reply:
Finally deleted in your commit. I thing it was mistake when i was fighting with ssr and csr hydrating.